script/bootstrap

```diff
@@ -22,7 +22,12 @@ test -d $VENV || python3 -m venv $VENV || return
 python -m pip install -U pip --no-cache-dir
 
 if [[ -z $SKIP_DEPS ]]; then
-    python -m pip install -e . -r dev_requirements.txt
+    if [[ -z $CI ]]; then
+        export PYTHONPATH="./src:$PYTHONPATH"
+    else
+        python -m pip install -e .
```

Comment:
dev_requirements.txt includes the install_requires. You can change this to `install -e . --no-deps`

Reply:
```suggestion
        python -m pip install -e . --no-deps
```